product-service/src/db/models/product.js

```diff
@@ -0,0 +1,10 @@
+import Joi from 'joi';
+
+const productSchema = Joi.object({
+  id: Joi.string(),
+  title: Joi.string().required(),
+  description: Joi.string(),
+  price: Joi.number().positive().allow(0),
```

Comment:
I think it is better to update price validation:
{
"id": "04009d9d-5288-4e17-93d8-63daf33a5609",
"title": "Deer",
"description": "Bamby",
"price": null,
"count": 0
},

Reply:
Thank you! Fixed.